### ISSUE TITLE
Feature Request: Dimention restoration while resizing on `esc` press

### ISSUE THREAD

**Issue by brian6932** [RamonUnch/AltSnap#504]
Basically, when you resize a window normally, pressing escape while resizing restores the window to it's dimensions before you started resizing, however AltSnap doesn't abide by this behavior. I think it'd be nice for this to be a toggleable feature within the `ini` file.

**Comment by RamonUnch** [comment]
Yes, this is because ESC is handled like a hard reset instead of a movement cancellation, so that if AltSnap locks something or gets in a funny state you can press ESC to fix it.
Because of that it tries not to do any extra work.
If you want the window to stay still when pressing ESC you can always use `FullWin=0` in the `[Performance]` section of the ini file, and it will move a hollow square instead of the window until movement is finished.

**Comment by brian6932** [comment]
Yea I figured it was due to the motion stop, what I propose is more specifically on resize
